drivers/modem/Kconfig.ehs6

```diff
@@ -0,0 +1,65 @@
+# Kconfig - Gemalto EHS6 driver options
+
+#
+# Copyright (c) 2018 Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#CONFIG_OFFLOAD_SOCKET
+
+menuconfig MODEM_EHS6
+	bool "Enable Gemalto EHS6 modem driver"
+	depends on MODEM
+	select MODEM_RECEIVER
+	select NET_OFFLOAD
+	select UART_MCUX_2 if BOARD_FRDM_K64F
+	help
+	  Choose this setting to enable Wistron WNC-M14A2A LTE-M modem driver.
+	  NOTE: Currently the pin settings only work with FRDM K64F shield.
+
+if MODEM_EHS6
+
+config MODEM_EHS6_RX_STACK_SIZE
+	int "Size of the stack for the WNC-M14A2A modem driver RX thread"
+	depends on MODEM_EHS6
+	default 1028
+	help
+	  This stack is used by the EHS6 RX thread.
+
+config MODEM_EHS6_APN_NAME
+	string "APN name for establishing network connection"
+	depends on MODEM_EHS6
+	default "bjlenovo07.xfdz.njm2mapn"
+	help
+	  This setting is used in the AT%PDNSET command to set the APN name
+	  for the network connection context.  Normally, don't need to change
+	  this value.
```

Comment:
I mentioned "Normally, don't need to change this value" in the Wistron modem settings, because it's ordered through the AT&T store with an AT&T IoT SIM.  This may not be true for your modem.